<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>308</width>
    <height>328</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FFmpeg Plugin Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>178</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Formats</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="wmaCheckBox">
        <property name="text">
         <string>Windows Media Audio</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="apeCheckBox">
        <property name="text">
         <string>Monkey's Audio (APE)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ttaCheckBox">
        <property name="text">
         <string>True Audio</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="aacCheckBox">
        <property name="text">
         <string>ADTS AAC</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="mp3CheckBox">
        <property name="text">
         <string>MP3 (MPEG audio layer 3)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="mp4CheckBox">
        <property name="text">
         <string>MPEG-4 AAC/ALAC</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="raCheckBox">
        <property name="text">
         <string>RealAudio 1.0/2.0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="shCheckBox">
        <property name="text">
         <string>Shorten</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ac3CheckBox">
        <property name="text">
         <string>AC3/EAC</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="dtsCheckBox">
        <property name="text">
         <string>DTS/DTS-Core</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="mkaCheckBox">
        <property name="text">
         <string>Matroska Audio (Dolby TrueHD Lossless)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="vqfCheckBox">
        <property name="text">
         <string>VQF</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>214</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel">
     <x>103</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>269</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>148</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
